config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "public": {
+    "default": {
+      "commerce-core-endpoint": "https://www.aemshop.net/graphql",
+      "commerce-endpoint": "https://na1-sandbox.api.commerce.adobe.com/Fwus6kdpvYCmeEdcCX7PZg/graphql",
+      "headers": {
+        "cs": {
+          "Magento-Customer-Group": "b6589fc6ab0dc82cf12099d1c2d40ab994e8410c",
+          "ac-environment-id": "Fwus6kdpvYCmeEdcCX7PZg",
+          "ac-scope-locale": "en-US",
+          "ac-price-book-id": "west_coast_inc"
```

Comment:
@sirugh @shiftedreality  Can we include `ac-channel-id` in the template?  Do we need to have `pricebook` included.